data_dir/nemesis.yml

```diff
@@ -0,0 +1,253 @@
+- disrupt_abort_repair:
```

Comment:
How would `nemesis.yml` file map with `nemesis_classes.yml`? Or is there no need for that?

Reply:
There isn't a need for that, so I decided to keep them decoupled.

Reply:
You need the nemesis_classes.